deepmd/dpmodel/fitting/general_fitting.py

```diff
@@ -85,6 +87,7 @@ def __init__(
         resnet_dt: bool = True,
         numb_fparam: int = 0,
         numb_aparam: int = 0,
+        bias_atom_e: Optional[np.ndarray] = None,
```

Comment:
we may rename all `bias_atom_e` to `bias_atom`..

Reply:
I will update this in the following PR.